### DISCUSSION TITLE
Space bar swipe and top generic row coloring

### DISCUSSION THREAD

**Discussion by ketabruce** [AnySoftKeyboard/AnySoftKeyboard#3033] [Q&A]
Hi!

I'm using a really old version of AnySoftKeyboard (v111 - 20130919).
There I have a dark grey backgrounded top generic row (with numbers).
With the "new" version (v1.10-r4) I'm not able to choose a dark numbered theme (I'm using ICS - Android 4.0 theme now),
Is there a way to edit some files to achieve this functionality?

My other question is swipe on space bar.
With the above version when I changed keyboard with swiping space bar no spaces were entered into the text are, but now with my current version swipe count as keypress. Is there a way not to enter space character on swipe action?

Thanks

**Comment by MrGoga** [comment]
Hi,

I have the same bug wiht ASK v1.10.1786 (6956), (version on F-Droid.org @ 2021-05-15)

I've discovered this bug is not only about inserting unnecessary space while changing keyboards using swipe on Space Bar but this unnecessary space insertion happens with any assigned swipe action to Space Bar.

I've created a main ticket about this issue:
**"Unnecessary space insertion while using swipe functionality for the Space Bar (any assigned swipe action)"** https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/3043

Let's see when developers will be able to fix this. For me it is quite major issue.
